lib/config/rules/functionality.js

```diff
@@ -72,7 +72,7 @@ module.exports = {
   // Require a `default` case in all `switch` statements.
   'switch-default': false,
   // Requires `===` and `!==` in place of `==` and `!=`.
-  'triple-equals': true,
+  'triple-equals': [true, 'allow-null-check'],
```

Comment:
Now that I think about it. I guess this could've been a complete branch off from master than the `tslint-react` or `house-keeping` branch.